src/haddock/modules/refinement/flexref/__init__.py

```diff
@@ -1,4 +1,16 @@
-"""Flexible refinement with CNS."""
+"""Flexible refinement with CNS.
+
+The `[flexref]` module (also known as `it1` stage in HADDOCK2.X series),
+is a semi-flexible simulated annealing (SA) protocol in torsion angle space.
+
+This semi-flexible SA consists of several stages:
+- High temperature rigid body search
+- Rigid body SA
+- Semi-flexible SA with flexible side-chains at the interface
+- Semi-flexible SA with fully flexible interface (both backbone and side-chains)
+
```

Comment:
By default, only the interface regions is treated as flexible. It is automatically defined based on the intermolecular contacts made. It is also possible to manually define the semi-flexible regions, and also define fully flexible regions that are allowed to move throughout the entire protocol from the high temperature rigid body molecular dynamics on.